src/PreflightTest.ts

```diff
@@ -585,14 +585,72 @@ function parseTimeMeasurement(nativeTimeMeasurement: {
 /**
  * Parse native call quality enum.
  */
-function parseCallQuality(
+function parseCallQuality(nativeCallQuality: any) {
+  switch (Platform.OS) {
+    case 'android': {
+      return parseCallQualityAndroid(nativeCallQuality);
+    }
+    case 'ios': {
+      return parseCallQualityIos(nativeCallQuality);
```

Comment:
Are the differences between ios and android documented somewhere? I understand RN customers don't need to care. But curious for native.